### ISSUE TITLE
Issues regarding the design of experimental methods

### ISSUE THREAD

**Issue by jsongliang** [CSUBioGroup/DeepDTAGen#9]
Dear Professor:
I have some questions I'd like to ask.
Why didn't you set an early stopping mechanism in the DTA task in your code? Your dataset is only divided into training set and test set, without a validation set. Did you directly test on the test set after training on the training set?
Isn't the general approach to first train on the training set, then find the best parameters on the validation set, and finally test on the test set?

Looking forward to your answer!
